### PR TITLE
V1.1.0: Add support for CSMs and chemical attribute search 

### PR DESCRIPTION
**Pull request by piehld**
## v1.1.0 (2023-06-20)

- Added support for including computed structure models (CSMs) in search (via `results_content_type`)
- Added support for chemical attribute search
- Documentation and code cleanup

### CODE REVIEW

File: tests/testsearch.py [hunk 1]
```diff
@@ -390,6 +390,86 @@ def testLargePagination(self):
             ok = False
         self.assertTrue(ok)
```
Comment (brindakv) [inline tests/testsearch.py:392]: There are some typos in this file on lines 354, 368, 374 and 380.

File: tests/testsearch.py [hunk 2]
```diff
@@ -390,6 +390,86 @@ def testLargePagination(self):
...
+        logger.info(result)
+        logger.info("result of second query:")
+        logger.info(result2)
+        self.assertTrue(ok)
+        resultL = list(result2())
+        ok2 = "6FJH" in resultL
+        self.assertTrue(ok2)
+        logger.info("Chemical Search Fluent Syntax: result length: (%d), ok: (%r), ok2: (%r)", len(resultL), ok, ok2)
+
+    def testMismatch(self):
```
Comment (brindakv) [inline tests/testsearch.py:421]: Perhaps add info on what this test does (similar to the other). 

Is there a test for mismatched type and attribute?

**Review by brindakv — APPROVED** [review]
@piehld @santiblaumann @MoonWalker30 Looks good. 

I made a few minor comments.

File: tests/testsearch.py (hunk 2 above)
Comment (santiblaumann) [reply tests/testsearch.py:421]: Yes - this is what testMismatch does.

Comment (brindakv) [reply tests/testsearch.py:421]: @santiblaumann Thanks. I see the `rcsb_chem_comp.name` and `STRUCTURE_ATTRIBUTE_SEARCH_SERVICE` combination.

**Comment by piehld** [comment]
Thanks @brindakv and @santiblaumann. I've fixed the typos and will merge now.

FYI @MoonWalker30, you may need to rebase with these changes.

**Pull request merged**